OpenEdXMobile/src/main/java/org/edx/mobile/inapppurchases/ProductManager.kt

```diff
@@ -0,0 +1,18 @@
+package org.edx.mobile.inapppurchases
+
+import org.edx.mobile.model.course.CourseComponent
+
+// Todo Remove when course product mapping is complete
+object ProductManager {
+    private val skuManager = mapOf(
+        "course-v1:edX+DemoX+Demo_Course" to "org.edx.mobile.integrationtest",
+        "course-v1:edX+Test101+course" to "org.edx.mobile.test_product",
+        "course-v1:DemoX+PERF101+course" to "org.edx.mobile.test_product1",
+        "course-v1:test2+2+2" to "org.edx.mobile.test_product2",
+        "course-v1:test3+3+3" to "org.edx.mobile.test_product3"
+    )
+
+    fun getProductByCourseId(courseId: String) = skuManager[courseId]
+
+    fun getProductByCourse(course: CourseComponent)= skuManager[course.courseId]
```

Comment:
better to have only one method to get `ProductId`, my suggestion is `getProductByCourseId`.